modules/swagger-codegen/src/main/resources/rust/model.mustache

```diff
@@ -0,0 +1,57 @@
+{{>partial_header}}
+{{#models}}
+{{#model}}
+{{#description}}
+/// {{{classname}}} : {{{description}}}
+{{/description}}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct {{classname}} {
+{{#vars}}
+  {{#description}}
+  /// {{{description}}}
+  {{/description}}
+  #[serde(rename = "{{baseName}}")] {{name}}: {{^required}}Option<{{/required}}{{^isPrimitiveType}}{{^isContainer}}super::{{/isContainer}}{{/isPrimitiveType}}{{{datatype}}}{{^required}}>{{/required}}{{#hasMore}},{{/hasMore}}
+{{/vars}}
+}
+
+impl {{classname}} {
+  {{#description}}
+  /// {{{description}}}
+  {{/description}}
+  pub fn new({{#requiredVars}}{{name}}: {{{datatype}}}{{^-last}}, {{/-last}}{{/requiredVars}}) -> {{classname}} {
+    {{classname}} {
+      {{#vars}}
+      {{name}}: {{#required}}{{name}}{{/required}}{{^required}}{{#isListContainer}}None{{/isListContainer}}{{#isMapContainer}}Hash:new(){{/isMapContainer}}{{^isContainer}}None{{/isContainer}}{{/required}}{{#hasMore}},{{/hasMore}}
+      {{/vars}}
+    }
+  }
+
+  {{#vars}}
+  pub fn set_{{name}}(&mut self, {{name}}: {{{datatype}}}) {
```

Comment:
Same as above, if it's an array, `super::` needs to be added to the child struct name.

Also, `super::` is generally missing if it's a non-primitive type.

Can we rewrite literally any use of a model into an absolute path? E.g. for model T, it should be referenced as `::models::{{classnameFile}}::{{classname}}`. Then we don't need to deal with super anywhere.